.github/workflows/sonar.yaml

```diff
@@ -1,6 +1,6 @@
 name: SonarCloud Build
 env:
-  SHOWCASE_VERSION: 0.26.1
+  SHOWCASE_VERSION: 0.27.0
```

Comment:
Five version update locations. Oof. We should find a way to refactor this to reduce the chance we miss one in the future.

Reply:
Yep, agreed. Created an issue about automating the showcase version (https://github.com/googleapis/gapic-generator-java/issues/1601), but also worth looking into refactoring so that the version is managed in one place.